### ISSUE TITLE
Add support for aliases

### ISSUE THREAD

**Issue by shindler** [unlight/sublime-import-helper#54]
In typescript it's possible to use aliases to ref. files. It would be nice if it would be possible to configure import helper to use them (even if separate configuration would be required) then whenever an import statement is going to contain double-dot (..) and lives under alias root directory we get a path based on the alias. For instance:

we have an angular app and an alias "@app" which points to "src/app". When trying to import in src/app/something/something/file.ts something from src/app/example.ts we should end up with @app/example instead of ../../example.

Do you think that this could be added to your plugin?

**Comment by unlight** [comment]
#52 Duplicate

**Comment by unlight** [comment]
> Do you think that this could be added to your plugin?

Everything is possible. The problem is how it should look in UI, because paths can be resolved in several ways:
`../../foo.component` can be same as:
1. `@app/components/foo.component.ts`
2. `~components/foo.component.ts`
3. other

What should be chosen and why...

**Comment by shindler** [comment]
Actually I gave the answer already in my proposal :): `whenever an import statement is going to contain double-dot` -> use alias.

However I think that this could be tuned up a little bit by adding an ability to select `alias-usage-strategy`:
- `always` then each import should be added using first matching alias
- `parent-only` only when import will contain double-dot (so we need to go up in directories structure)

**Comment by unlight** [comment]
@shindler @jurito 
Please provide your `tsconfig.json` contents

**Comment by shindler** [comment]
@unlight 
```
{
  "compileOnSave": false,
  "compilerOptions": {
    "outDir": "./dist/out-tsc",
    "baseUrl": "./src",
    "sourceMap": true,
    "declaration": false,
    "moduleResolution": "node",
    "emitDecoratorMetadata": true,
    "experimentalDecorators": true,
    "preserveConstEnums": true,
    "alwaysStrict": true,
    "target": "es5",
    "typeRoots": [
      "node_modules/@types"
    ],
    "paths": {
      "@corpo-ui": ["corpo-ui/lib"],
      "@portal/*" :["app/*"]
    },
    "lib": [
      "es2017",
      "dom"
    ],
    "plugins": [
      {"name": "@angular/language-service"}
    ]
  }
}
```

here you go, my `tsconfig.json` used within angular-cli application. First alias `@corpo-ui` is for loading module, second allows to load anything from given location... so for example inside `src/app/core/services/some.service.ts` we can do `import Something from '@portal/configs/something.config.ts'` instead of `import Something from '../../configs/something.config.ts'` assuming that `configs` directory lives under `src/app/`.

**Comment by unlight** [comment]
Implemented in v2.0.0+
https://github.com/unlight/sublime-import-helper#import_path_mapping
But is not enabled by default.
